docs/csharp/misc/cs0077.md

```diff
@@ -13,6 +13,8 @@ ms.assetid: 55d3d290-d172-41a3-b326-ebf5a0a7e81f
 The as operator must be used with a reference type or nullable type ('int' is a non-nullable value type).  
   
  The [as](../language-reference/operators/type-testing-and-cast.md#as-operator) operator was passed a [value type](../language-reference/builtin-types/value-types.md). Because `as` can return [null](../language-reference/keywords/null.md), it can only be passed a [reference type](../language-reference/keywords/reference-types.md) or a [nullable value type](../language-reference/builtin-types/nullable-value-types.md).
+
+However, using [pattern matching](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/pattern-matching) with [is](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/is) operator, we can directly perform type checking and assignments in one step.
```

Comment:
Use relative paths if possible:
```suggestion
However, using [pattern matching](../fundamentals/functional/pattern-matching) with [is](../language-reference/operators/is) operator, we can directly perform type checking and assignments in one step.
```
This will also fix the CI concerns.